Source/Mosa.DeviceSystem/HAL.cs

```diff
@@ -54,7 +54,7 @@ public static void ProcessInterrupt(byte irq)
 		/// </summary>
 		/// <param name="port">The port number.</param>
 		/// <returns></returns>
-		internal static BaseIOPortReadWrite GetReadWriteIOPort(ushort port)
+		public static BaseIOPortReadWrite GetReadWriteIOPort(ushort port)
```

Comment:
Not sure about this. It breaks the abstraction. But I understand the dilemma here.

Reply:
It compiles fine and runs fine, so I guess we can keep it (I mean I don't really have a choice either way :/)